.github/workflows/homework_04_02_angel_stoyanov.yaml

```diff
@@ -0,0 +1,46 @@
+name: Homework 04 - 02 Angel Stoyanov
```

Comment:
The only problem I see in the YAML file is the convention. Your names should start with lower case - here and below on lines 6, 12, 37 and 42. Everything else here seems okay for me. Excellent!